make packer fails

Here is what happens:

```
$ make packer
docker run \
        -e AWS_DEFAULT_REGION  \
        -e AWS_PROFILE \
        -e AWS_ACCESS_KEY_ID \
        -e AWS_SECRET_ACCESS_KEY \
        -e AWS_SESSION_TOKEN \
        -e PACKER_LOG \
        -v /home/mark/.aws:/root/.aws \
        -v "/home/mark/projects/buildkitet/elastic-ci-stack-for-aws:/src" \
        --rm \
        -w /src/packer/linux \
        hashicorp/packer:1.0.4 build -var 'ami=ami-0b898040803850657' -var 'region=us-east-1' \
                buildkite-ami.json | tee packer-linux.output
amazon-ebs output will be in this color.

==> amazon-ebs: Prevalidating AMI Name...
    amazon-ebs: Found Image ID: ami-0a887e401f7654935
==> amazon-ebs: Creating temporary keypair: packer_5e67946e-9d8d-3551-7d58-82898385d551
mark@fou ~/p/b/elastic-ci-stack-for-aws> make packer
docker run \
        -e AWS_DEFAULT_REGION  \
        -e AWS_PROFILE \
        -e AWS_ACCESS_KEY_ID \
        -e AWS_SECRET_ACCESS_KEY \
        -e AWS_SESSION_TOKEN \
        -e PACKER_LOG \
        -v /home/mark/.aws:/root/.aws \
        -v "/home/mark/projects/buildkitet/elastic-ci-stack-for-aws:/src" \
        --rm \
        -w /src/packer/windows \
        hashicorp/packer:1.0.4 build -var 'region=us-east-1' \
                buildkite-ami.json | tee packer-windows.output
amazon-ebs output will be in this color.

==> amazon-ebs: Prevalidating AMI Name...
    amazon-ebs: Found Image ID: ami-062b1528dfe4a5f66
==> amazon-ebs: Creating temporary keypair: packer_5e679bd9-186b-54da-83ea-37e7d0502c9d
==> amazon-ebs: Creating temporary security group for this instance: packer_5e679bdc-a920-452f-3aec-14110fd8d609
==> amazon-ebs: Authorizing access to port 5986 on the temporary security group...
==> amazon-ebs: Launching a source AWS instance...
    amazon-ebs: Finding spot price for Windows (Amazon VPC) m5.xlarge...
    amazon-ebs: Requesting spot instance 'm5.xlarge' for: 0.257
==> amazon-ebs: Error launching source spot instance: InvalidSubnetID.NotFound: No default subnet for availability zone: 'us-east-1d'.
==> amazon-ebs:         status code: 400, request id: 3c49dda2-101f-4655-8e5e-f26fb1396e0a
==> amazon-ebs: No volumes to clean up, skipping
==> amazon-ebs: Deleting temporary security group...
==> amazon-ebs: Deleting temporary keypair...
Build 'amazon-ebs' errored: Error launching source spot instance: InvalidSubnetID.NotFound: No default subnet for availability zone: 'us-east-1d'.
        status code: 400, request id: 3c49dda2-101f-4655-8e5e-f26fb1396e0a

==> Some builds didn't complete successfully and had errors:
--> amazon-ebs: Error launching source spot instance: InvalidSubnetID.NotFound: No default subnet for availability zone: 'us-east-1d'.
        status code: 400, request id: 3c49dda2-101f-4655-8e5e-f26fb1396e0a

==> Builds finished but no artifacts were created.
make: *** [Makefile:90: packer-windows.output] Error 1
```

Other notes:

* Shell could be `SHELL = bash -o pipefail` instead of using `/bin/bash` which is not always available (I'm running this on NixOS).

--- a/README.md
+++ b/README.md
@@ -61,11 +61,19 @@ AWS_PROFILE="some-profile" make create-stack
 aws-vault exec some-profile -- make create-stack
 ```
 
-If you need to build your own AMI (because you've changed something in the `packer` directory), run:
+If you need to build your own AMI (because you've changed something in the
+`packer` directory), run packer with AWS credentials in your shell environment:
 
 ```bash
 make packer
 ```
+
+This will boot and image three AWS EC2 instances in your account’s `us-east-1`
+default VPC:
+
+- Linux amd64
+- Linux arm64
+- Windows amd64
 
 ## Support Policy
 

